### PR TITLE
Tiny improvements for the text tutorial

### PR DESCRIPTION
**Pull request by core-man**
**Description of proposed changes**

Some tiny improvements for the tutorial for the text.

Related to #983 and #996.

**Reminders**

- [ ] Run `make format` and `make check` to make sure the code follows the style guide.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Add new public functions/methods/classes to `doc/api/index.rst`.
- [ ] Write detailed docstrings for all functions/methods.
- [ ] If adding new functionality, add an example to docstrings or tutorials.

**Slash Commands**

You can write slash commands (`/command`) in the first line of a comment to perform
specific operations. Supported slash commands are:

- `/format`: automatically format and lint the code
- `/test-gmt-dev`: run full tests on the latest GMT development version

### CODE REVIEW

**Comment by seisman** [comment]
Some minor comments:

1. Line 64: change "recognised" to "recognized" (American spelling)
2. Line 65: change <code>:gmt-docs:\`cookbook/postscript-fonts.html\`</code> to <code>
:gmt-docs:\`PostScript Fonts Used by GMT <cookbook/postscript-fonts.html>\` </code>

**Comment by core-man** [comment]
> Some minor comments:
> 
> 1. Line 64: change "recognised" to "recognized" (American spelling)
> 2. Line 65: change `` :gmt-docs:`cookbook/postscript-fonts.html` `` to `` 
>    :gmt-docs:`PostScript Fonts Used by GMT <cookbook/postscript-fonts.html>`  ``

done in a4121e4.

File: examples/tutorials/text.py [hunk 1]
```diff
@@ -152,4 +152,4 @@
 #
 # For crafting more advanced styles, be sure to check out the GMT documentation
 # at :gmt-docs:`text.html` and also the cookbook at
-# :gmt-docs:`cookbook/features.html#placement-of-text`. Good luck!
+# :gmt-docs:`PostScript Fonts Used by GMT <cookbook/postscript-fonts.html>`. Good luck!
```
Comment (seisman) [inline examples/tutorials/text.py:155]: You made wrong changes here. I meant to change the reference at line 55, not here.

Comment (core-man) [reply examples/tutorials/text.py]: Fixed in ec4e401.

**Pull request merged**